db/db_manager.py

```diff
@@ -137,28 +137,28 @@ def update_user_info(
     def update_user_password(
             self,
             user_id: str,
-            new_password: str,
-            old_password: str
-    ) -> Optional[ObjectId]:
+            old_password: str,
+            new_password: str
+    ) -> Tuple[Dict[str, str], int]:
         """ Update the user's password as hash """
         users = self._db['users']
         user = users.find_one({'_id': ObjectId(user_id)})
 
         if not user:
-            return None
```

Comment:
Leave it as None the dict and status code should be handled in the routes

Reply:
Fixed